test/system/accounts_test.rb

```diff
@@ -0,0 +1,18 @@
+require "application_system_test_case"
+
+class AccountsTest < ApplicationSystemTestCase
+  setup do
+    sign_in @user = users(:bob)
```

Comment:
A pattern I see often and we could consider `sign_in 'bob'`  or `sign_in users(:bob)`

```
def sign_in(user)
  user = users(user) unless user.is_a? User
  ....
end
```

Gives a little more flexibility in usage

Reply:
This is really cute. I only saw it recently for the first time in Campfire.

Reply:
same